src/pulselistener/pulselistener/listener.py

```diff
@@ -110,7 +110,7 @@ def list_differential(self):
                     logger.info('Skipping Taskcluster task', diff=diff['phid'])
 
                 # Put message in mercurial queue for try jobs
-                if ACTION_TRY in self.actions:
+                if ACTION_TRY in self.actions and self.mercurial_queue:
                     await self.mercurial_queue.put(diff)
```

Comment:
Maybe assert if ACTION_TRY and mercurial_queue is None?

Reply:
Good point, i updated it